vike/node/cli/commands/eject.ts

```diff
@@ -0,0 +1,62 @@
+import { resolve } from 'path'
+import fs from 'fs'
+
+export function eject(dependencies: string[]) {
```

Comment:
I left this here for reference, I can clean it up if we don't need it. The actual requested command is the `scaffold` command